web/impact/impact/urls.py

```diff
@@ -38,23 +42,22 @@
     UserListView,
     UserOrganizationsView,
 )
-from rest_framework import routers
-from drf_auto_endpoint.router import router as schema_router
-from django.apps import apps
-
 
 accelerator_router = routers.DefaultRouter()
 simpleuser_router = routers.DefaultRouter()
 
 simpleuser_router.register('User', GeneralViewSet, base_name='User')
 
 for model in apps.get_models('impact'):
-    if model._meta.app_label == 'impact':
-        schema_router.register(model, url=model.__name__)
-
+    if model._meta.app_label == 'impact' and not model._meta.auto_created:
+        schema_router.register(model, url=model_name_to_snake(model.__name__))
 
 for model in apps.get_models('accelerator'):
-    if model._meta.app_label == 'accelerator' and hasattr(model, "Meta"):
+    if (
+        model._meta.app_label == 'accelerator'
+        and hasattr(model, "Meta")
+        and not model._meta.auto_created
+    ):
```

Comment:
I don't love the hanging `):`
Can we remove the line breaks at 56 and 59?

Reply:
See the previous discussion.  I will let you and @yotammanor debate this.

Reply:
Ideally, complex conditionals would be named for the condition we actually care about and turned into functions. Part of the problem for me in this code review is I have no idea what the purpose of this conditional is - what does it mean when a model has a `.meta.app_label` of `'accelerator'` and has an attribute called `Meta` and meta says it's auto-created? 

Since we just want to get this merged, I'm not going to push this point, but our house style (many examples) for a complex conditional looks like this:

```
    if (model._meta.app_label == 'accelerator' and
        hasattr(model, "Meta") and
       not model._meta.auto_created):
```

Reply:
Now that I look at that list of conditions, though - does `not hasattr(model, 'Meta')` imply that the lookup `model._meta` will fail? I don't know off hand, but it seems safer to do the hasattr check first.